iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/topology/MoveFeederBayTest.java

```diff
@@ -23,39 +23,179 @@ class MoveFeederBayTest {
+        int initialNode = line.getTerminal1().getNodeBreakerView().getNode();
+
+        // Move the branch terminal
+        new MoveFeederBayBuilder()
+                .withConnectableId("line1")
+                .withTargetBusOrBusBarSectionId("bbs5")
+                .withTargetVoltageLevelId("vl2")
+                .withTerminal(line.getTerminal1())
+                .build()
+                .apply(network);
+
+        // Verify move
+        assertEquals("vl2", line.getTerminal1().getVoltageLevel().getId());
+        assertFalse(line.getTerminal1().isConnected());
+        int movedNode = line.getTerminal1().getNodeBreakerView().getNode();
+        assertNotEquals(initialNode, movedNode);
+
+        // Verify terminal2 is unchanged
+        assertEquals("vlSubst2", line.getTerminal2().getVoltageLevel().getId());
     }
 
     @Test
     void testMoveInjectionBusBarBreaker() {
         Network network = EurostagTutorialExample1Factory.create();
-        new MoveFeederBayBuilder().withConnectableId("GEN").withBusOrBusbarSectionId("NHV1").withVoltageLevelId("VLLOAD")
-                .withTerminal(network.getGenerator("GEN").getTerminal()).build().apply(network);
 
+        // Verify initial state
+        Generator generator = network.getGenerator("GEN");
+        assertEquals("VLGEN", generator.getTerminal().getVoltageLevel().getId());
+        String initialBusId = generator.getTerminal().getBusBreakerView().getBus().getId();
+        boolean initialConnected = generator.getTerminal().isConnected();
+
+        // Move generator
+        new MoveFeederBayBuilder()
+                .withConnectableId("GEN")
+                .withTargetBusOrBusBarSectionId("NHV1")
+                .withTargetVoltageLevelId("VLHV1")
+                .withTerminal(generator.getTerminal())
+                .build()
+                .apply(network);
+
+        // Verify move
+        assertEquals("VLHV1", generator.getTerminal().getVoltageLevel().getId());
+        assertEquals("NHV1", generator.getTerminal().getBusBreakerView().getBus().getId());
+        assertNotEquals(initialBusId, generator.getTerminal().getBusBreakerView().getBus().getId());
+        assertEquals(initialConnected, generator.getTerminal().isConnected());
     }
 
     @Test
     void testMoveBbsBreaker() {
         Network network = Network.read("testNetworkNodeBreaker.xiidm", getClass().getResourceAsStream("/testNetworkNodeBreaker.xiidm"));
-        ReportNode reportNode = ReportNode.newRootReportNode().withMessageTemplate("reportTestMoveBbs", "Testing reportNode when trying to remove a busbar section").build();
-        MoveFeederBay moveFeederBay = new MoveFeederBayBuilder().withConnectableId("bbs6").withBusOrBusbarSectionId("bbs5").withVoltageLevelId("vl1")
-                .withTerminal(network.getLine("line1").getTerminal1()).build();
+        ReportNode reportNode = ReportNode.newRootReportNode()
+                .withMessageTemplate("reportTestMoveBbs")
```

Comment:
With ReportNode API V3, you need to specify the bundle to use when a new root node is created:
```suggestion
        ReportNode reportNode = ReportNode.newRootReportNode()
                .withResourceBundles(PowsyblCoreTestReportResourceBundle.TEST_BASE_NAME, PowsyblCoreReportResourceBundle.BASE_NAME)
                .withMessageTemplate("reportTestMoveBbs")
```